test_demo_any.py

```diff
@@ -0,0 +1,33 @@
+'''
+The script does inference (semantic segmentation) on arbitrary images.
+Just drop some JPG files into demo_dir and run the script.
+Results will be written into the same folder.
+For better results channel_means better be recalculated I suppose. But it is kinda tricky.
+'''
+from os import path as osp
+from glob import glob
+import numpy as np
+
+from model import DeepLab
+from utils import ( save_load_means, subtract_channel_means, single_demo, read_image)
+
+if __name__ == '__main__':
+
+    demo_dir = 'data/demos/deeplab/resnet_101_voc2012/'
```

Comment:
The best way to do this with some customized arguments is probably to use `argparse`. Check some examples such as my `train.py`.